src/CommunityToolkit.Maui.UnitTests/Converters/StringToListConverter_Tests.cs

```diff
@@ -19,30 +19,40 @@ public class StringToListConverter_Tests : BaseTest
 
 	[Theory]
 	[MemberData(nameof(ListData))]
-	public void StringToListConverter(object? value, object? parameter, object? expectedResult)
+	public void StringToListConverter(string? value, object? parameter, object? expectedResult)
 	{
 		var stringToListConverter = new StringToListConverter();
 
-		var result = (IEnumerable<string>)stringToListConverter.Convert(value, typeof(IEnumerable<string>), parameter, null);
+		var convertFromResult = stringToListConverter.ConvertFrom(value, typeof(IList<string>), parameter, null);
```

Comment:
Good question! 

I initially made `StringToListConverter` require an `IList` (mostly because its name contains the word `List`. But, after some testing, I lowered it from `IList` to `IEnumerable`.

That being said, I think it's best to keep this Unit Test as `IList` because it demonstrates/tests the updated validation method of  `BaseConverter`:
```cs
if (!typeof(TTo).IsAssignableFrom(targetType) && !IsValidTargetType<TTo>(targetType))
	throw new ArgumentException($"targetType needs to be assignable from {typeof(TTo)}", nameof(targetType));
```

In this unit test, `typeof(List<string>).IsAssignableFrom(IEnumerable)` evaluates to `true`.